src/plugins/browser-console/publisher.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseEventModel, BasePublisher } from '../../base';
+import { StratumSnapshot } from '../../types';
+
+/**
+ * Browser Console Publisher
+ */
+export class BrowserConsolePublisher extends BasePublisher {
+  name = 'browserConsole';
+
+  /**
+   * Handle all event types generically, even those provided
+   * by a separate plugin.
+   */
+  shouldPublishEvent(): boolean {
+    return true;
+  }
+
+  /**
+   * Required
+   * Returns data from underlying events to utilize
+   * in the publish step.
+   */
+  getEventOutput(_event: BaseEventModel, snapshot: StratumSnapshot) {
+    return snapshot.data;
+  }
+
+  /**
+   * Required
+   * Check if your publisher source is available (aka scripts installed, environment
+   * is set up, etc.)
+   *
+   * In this case, we make sure that console.log() is accessible.
+   */
+  async isAvailable() {
+    return typeof console !== 'undefined';
+  }
+
+  /**
+   * Required
+   * Send your content to the external publisher
+   *
+   * In this, case we publish the snapshot data to the console log
+   */
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+  async publish(_content: any, snapshot: StratumSnapshot) {
+    console.log(snapshot.data);
```

Comment:
I think we should send more data to the console as `snapshot.data` only contains the output from `model.getData()`. Other metadata like the tag ID or event type within the snapshot would also be useful.

Reply:
Updated! Changed the `getEventOutput` return value and updated the `publish` method accordingly. Thought that'd better show how they relate. It'll now publish to the console something like:
`BrowserConsolePlugin: {"eventType":"eventTypeName","id":1}`